plutus-ledger/changelog.d/20230412_134023_sjoerd.visscher_PLT_5294_replace_more_plutus_types.md

```diff
@@ -1,3 +1,7 @@
+### Removed
+
+- Removed `TxIn` and `TxInput` types.
```

Comment:
Maybe explain that we use Cardano types instead?

Reply:
That's not true actually. `TxInput` wasn't used at all anymore and the only place where `TxIn` was still used was in `ChainIndexTx` where I replaced it with `TxOutRef`. (The chain-index leans heavily on TxOutRef, and I have not replaced it there.)